### ISSUE TITLE
Errors initializing ESLint

### ISSUE THREAD

**Issue by jakefentress** [apexskier/nova-eslint#329]
ESLint never activates and runs when opening Nova or enabling the extension.

But eslint appears to run fine from the local terminal via `$(npm bin)/eslint ./_src/static/js/global.js`

**To reproduce**

1. Open Nova with ESLint extension enabled or enable the ESLint extension from the Extension Library
2. Do nothing else
3. Check the Extension Console
4. See output provided below

**Expected behavior**

When enabling the ESLint extension, it would lint js files without erroring on an uncaught exception.

**Screenshots**

![Screen Shot 2022-09-09 at 3 33 17 PM](https://user-images.githubusercontent.com/2132238/189438577-ba70a5e2-21a5-4201-8848-5df612c51a51.png)

With the extension enabled, no issues are found, even with obvious formatting errors in the open file.

![Screen Shot 2022-09-09 at 3 34 08 PM](https://user-images.githubusercontent.com/2132238/189438663-a19f6582-81e5-4fa3-b6a6-9f99a71b074b.png)

Running `$(npm bin)/eslint ./_src/static/js/global.js` does reveal the expected formatting error.

**Environment**

- Nova version: 9.5 (437303)
- Extension version: 1.7.3
- ESLint version: v7.32.0 (but also tested v8.23.0)
- macOS version: 12.5.1
- NodeJS information:
  - node version: v14.6.0 (but also tested v14.20.0 with the newer ESLint listed above)
  - npm version: 8.19.1
  - installation method: nvm

**Extension console output**

```
ESLint[15:25:41.584000] activating...

ESLint[15:25:41.873000] activated

ESLint[15:25:42.153000] Failed to lint

ESLint[15:25:42.153000]   stderr:  
Oops! Something went wrong! :(

ESLint: 7.32.0

TypeError: createRequire is not a function
Referenced from: /Users/USERNAME/Sites/projects/PROJECT/.eslintrc.json
    at Object.resolve (/Users/USERNAME/Sites/projects/PROJECT/node_modules/@eslint/eslintrc/lib/shared/relative-module-resolver.js:28:20)
    at ConfigArrayFactory._loadExtendedShareableConfig (/Users/USERNAME/Sites/projects/PROJECT/node_modules/@eslint/eslintrc/lib/config-array-factory.js:879:33)
    at ConfigArrayFactory._loadExtends (/Users/USERNAME/Sites/projects/PROJECT/node_modules/@eslint/eslintrc/lib/config-array-factory.js:781:25)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/Users/USERNAME/Sites/projects/PROJECT/node_modules/@eslint/eslintrc/lib/config-array-factory.js:720:25)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigData (/Users/USERNAME/Sites/projects/PROJECT/node_modules/@eslint/eslintrc/lib/config-array-factory.js:665:20)
    at _normalizeObjectConfigData.next (<anonymous>)
    at ConfigArrayFactory.loadFile (/Users/USERNAME/Sites/projects/PROJECT/node_modules/@eslint/eslintrc/lib/config-array-factory.js:475:16)
    at createCLIConfigArray (/Users/USERNAME/Sites/projects/PROJECT/node_modules/@eslint/eslintrc/lib/cascading-config-array-factory.js:171:35)
    at new CascadingConfigArrayFactory (/Users/USERNAME/Sites/projects/PROJECT/node_modules/@eslint/eslintrc/lib/cascading-config-array-factory.js:243:29)


ESLint[15:25:42.153000]   command:  /Users/USERNAME/Sites/projects/PROJECT/node_modules/.bin/eslint

ESLint[15:25:42.154000]   args:  --config /Users/USERNAME/Sites/projects/PROJECT/.eslintrc.json --format=json --stdin

ESLint[15:25:42.155000] Extension encountered an uncaught exception:
/Users/USERNAME/Library/Application Support/Nova/Extensions/apexskier.eslint/Scripts/main.dist.js (Line 377, Column 32)
Error: failed to lint (2)
    @file:///Users/USERNAME/Library/Application%20Support/Nova/Extensions/apexskier.eslint/Scripts/main.dist.js:377:32
    @[native code]
    promiseReactionJob@[native code]

```

**Comment by jakefentress** [comment]
After months, of course I find a solution immediately after submitting this issue.

Somehow Nova was locked to an old default version of Node (via NVM) instead of picking up the newer - 10.8.0 vs 14.16.0 - for extensions (the newer was used by the Nova local terminal window, however). After removing the old version and restarting Nova, everything worked as expected.

Closing this out and apologies for the noise. Hopefully this will help someone else.
